Marketplace/Providers/onboarding-resources.md

```diff
@@ -39,6 +39,6 @@ A CenturyLink Cloud Account is required to be a provider on the CenturyLink Cl
 
 ### Getting Started Guide: Marketplace & Ecosystem Provider Portal
 
-The [Marketplace Getting Started Guide](https://www.ctl.io/knowledge-base/marketplace/providers/getting-started-guide-marketpalce-ecosystem-provider-portal/) will serve as your primary guide for creating your product in the Provider Portal. We recommend using it in conjunction with the training videos provided within the Portal. Our goal for the CenturyLink Cloud Marketplace is to provide a superior user experience with respect to learning about your product. If you'd like additional support developing your content, we're here to help. Contact us at [Marketplace@ctl.io](mailto:marketplace@ctl.io).
```

Comment:
Change link to `./getting-started-guide-marketplace-ecosystem-provider-portal.md`